### PR TITLE
Bindings/props cleanup (element directives)

### PR DESCRIPTION
**Pull request by arxpoetica**

### CODE REVIEW

File: content/guide/02-template-syntax.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ If the expression in `{#await expression}` *isn't* a promise, Svelte skips ahead
 
 ### Directives
 
-The last place where Svelte template syntax differs from regular HTML: *directives* allow you to add special instructions for adding [event handlers](guide#event-handlers), [two-way bindings](guide#two-way-binding), [refs](guide#refs) and so on. We'll cover each of those in later stages of this guide – for now, all you need to know is that directives can be identified by the `:` character:
+The last place where Svelte template syntax differs from regular HTML: *directives* allow you to add special instructions for adding [event handlers](guide#event-handlers), [binding data](guide#binding-data), [refs](guide#refs) and so on. We'll cover each of those in later stages of this guide – for now, all you need to know is that directives can be identified by the `:` character:
```
Comment (jacwright) [inline content/guide/02-template-syntax.md]: While you're at it, could we change `[refs]` to `[referencing elements]`, the action verb which both matches the previous two and also makes sense to someone new to Svelte?

Comment (Rich-Harris) [reply content/guide/02-template-syntax.md]: I've made that edit

File: content/guide/05-nested-components.md [hunk 2]
```diff
@@ -49,25 +49,11 @@ const widget = new Widget({
...
-<Widget foo={foo}/>
-```
-
-you can use
-
-```html
-<!-- { repl: false } -->
-<Widget {foo}/>
-```
+For a detailed description of the relationship of data between components, see the section on [passing data](guide#passing-data) and [binding data](guide#binding-data).
```
Comment (Rich-Harris) [inline content/guide/05-nested-components.md]: I think we need to rethink this — props aren't directives, so they can't really go in the directives section. Lemme fiddle with it a bit

**Comment by Rich-Harris** [comment]
Made a few edits. Staged deployment: https://sveltetechnology-gtemhlkgaf.now.sh/guide#nested-components

**Comment by Conduitry** [comment]
Glad my embarrassing list of available bindings is getting replaced by a nice-looking table 😀 

However, pending https://github.com/sveltejs/svelte/issues/1104 we should probably indicate that `bind:checked` doesn't actually work on radio inputs.

**Comment by arxpoetica** [comment]
Since you moved props back up (I find that confusing, but I understand how it makes sense), can we at least include a link down in the directives section for quickly jumping between bindings and props? The main thing I'm always wanting to look up is the difference between the two, such as shorthands or principle of data flow.

**Comment by Rich-Harris** [comment]
Good catch @Conduitry, have updated that part and added a new example.

Thanks everyone, think this is a solid improvement on what we had.

**Pull request merged**